third_party/nvidia/backend/driver.py

```diff
@@ -288,6 +279,9 @@ def format_of(ty):
         PyErr_Format(PyExc_ValueError,
                      "Pointer argument (at %d) cannot be accessed from Triton (cpu tensor?)", idx);
         ptr_info.valid = false;
+    }} else if (status != CUDA_SUCCESS) {{
+        CUDA_CHECK(status);  // Catch any other cuda API errors
+        ptr_info.valid = false;
```

Comment:
The reason we get an invalid memory access error in the test is that `cuPointerGetAttributes` is returning `CUDA_ERROR_INVALID_CONTEXT` which wasn't being handled here and we end up in an invalid state.